README.md

```diff
@@ -7,32 +7,63 @@ The AWS Database Migration Service is used to migrate data from a database (RDS
 ## Pre-requirements
 
  1 - The _source_ database needs to have its _Public Accessibility_ settings turned on (or the equivalent firewall rule in other environments)
- 2 - For Postgrs-to-Postgres, the _source_ RDS's parameter group need to comply with :
+
+ 2 - For Postgres in RDS, the _source's_ parameter group need to comply with :
    - rds.logical_replication = 1
    - max_replication_slots > 5
 
  **DMS ONLY MIGRATES DATA, NO PRE-DATA, POST-DATA(CONSTRAINTS), USERS, ROLES, ETC.**
 
+ **Continuous replication, especially between different engines, is a tricky business, see https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Task.CDC.html for details**
+
+## Schema migration
+
+WIP
```

Comment:
If this isn't due to be part of this PR, I'd recommend removing this sub-heading until it's ready.